src/nnue/nnue_accumulator.h

```diff
@@ -29,8 +29,10 @@
 namespace Stockfish::Eval::NNUE {
 
 // Class that holds the result of affine transformation of input features
+template<bool Small>
```

Comment:
Why take this instead of the size?

Reply:
Because it's instantiated inside StateInfo in position.h which doesn't (and probably shouldn't) know anything about any accumulator sizes.

Reply:
It now takes size.  Turns out position.h already includes "nnue/nnue_accumulator.h" which I overlooked before.

Reply:
Sanitizer made me add #include "nnue/nnue_architecture.h" to position.h even though it compiles w/o.